AppCenter/AppCenter/MSAppCenter.h

```diff
@@ -38,6 +38,14 @@
  */
 + (void)startService:(Class)service;
 
+/**
+ * Configure the SDK with an array of services to start from a library.
+ * This will not start the service at application level, it will enable the service only for the library.
+ *
+ * @param services  Array of services to start.
```

Comment:
Nit,  there is two spaces between "services" and "Array".

Reply:
Good catch! 😄